homeassistant/components/google_assistant/trait.py

```diff
@@ -937,6 +971,43 @@ async def _execute_vacuum(self, command, data, params, challenge):
                     context=data.context,
                 )
 
+    async def _execute_lawn_mower(self, command, data, params, challenge):
+        """Execute a StartStop command."""
+        if command == COMMAND_START_STOP:
+            if params["start"]:
+                await self.hass.services.async_call(
```

Comment:
the 4 service calls are equal, except for the service being called. could we instead have a `service` variable and invoke `async_call` with it, to avoid duplicating the call 4 times?

Reply:
I copied the code from the vacuum. I pushed the changes.